services/frontend-service/src/ui/Pages/Home/Home.test.tsx

```diff
@@ -32,20 +32,278 @@ describe('App', () => {
     const getWrapper = () => render(getNode());
     it('Renders full app', () => {
         // when
+        const appBuilder = (suffix: string) => ({
+            name: 'test' + suffix,
+            releases: [],
+            sourceRepoUrl: 'http://test' + suffix + '.com',
+            team: 'example' + suffix,
+        });
         UpdateOverview.set({
             environments: {},
             applications: {
-                app1: {} as any,
-                app3: {} as any,
-                app2: {} as any,
+                app1: appBuilder('1'),
```

Comment:
Instead of calling the function appBuilder here, create an object (called for example SampleApps) and in there create the apps. Then in here just call the sample app that you want.

Reply:
An array of sample apps also works